python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -153,6 +153,8 @@ def from_cudf_edgelist(
             pylibcugraph algorithms.  Must be false for algorithms
             not yet converted to the pylibcugraph C API.
 
+            This parameter is deprecated and will be removed.
```

Comment:
Can you also raise a `DeprecationWarning` if the user sets `legacy_renum_only` to `True`?

Reply:
So this function eventually calls `self._Impl._simpleGraphImpl__from_edgelist` which intrinsically raises the `DeprecationWarning`. I didn't add an extra one to avoid duplicates